processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -222,10 +236,10 @@ private static ColumnCapabilities getEffectiveCapabilities(
     return strategyFactory.makeColumnSelectorStrategy(capabilities, selector);
   }
 
-  public static Long convertObjectToLong(Object valObj)
+  public static Long convertObjectToLong(@Nullable Object valObj)
```

Comment:
Could you comment on not handling null from getExactLongFromDecimalString(), that is not aligned with convertObjectToDouble()?